packages/kas/src/nodes.ts

```diff
@@ -3820,9 +3765,13 @@ const makeAlias = function (str: string, prefixes: Prefixes) {
     var coefficientStr = splits[0].trim();
     var unitsStr = splits[1].trim();
 
-    var coefficient = NumOne;
+    var coefficient: Expr = NumOne;
     if (coefficientStr !== "") {
-        coefficient = parse(coefficientStr).expr;
+        const result = parse(coefficientStr);
+        if (!result.parsed) {
+            throw new Error(result.error);
+        }
+        coefficient = result.expr;
```

Comment:
TODO: Revert this change and add a `@ts-expect-error`.  All of the calls to `makeAlias` happen on initialization.  Any issues should be caught by the unit tests.  I'll leave a comment about this.